### DISCUSSION TITLE
Legends gamer mini (and pro)

### DISCUSSION THREAD

**Discussion by Paradox** [retronas/discussions#96] [Ideas]
I picked up a legends gamer mini, today at a thrift store, and, while playing around with it, found that it allows you to stream games locally, but, it requires a special software run on windows as the server, which probably means it can't be used with RetroNAS, unless someone has figured out how to create a Linux version... I'm hoping someone has, or, can, so we can add it to RetroNAS. The name of the app is legends link app (formerly arcadenetlinkapp)

**Comment by Paradox** [comment]
I thought this was for arcade and retro roms, but, now that I have gone to the website, it's more like gog or steam...

**Comment by birdybro** [comment]
Looks like it's just the on-premise equivalent of their cloud game streaming software that you would pay monthly for, and it only supports some limited games that they have licensed.

https://www.atgames.net/arcadenet/byog

This is probably out of scope.

**Reply by Paradox** [reply]
Yeah, I didn't realize that, when I posted.

**Reply by Paradox** [reply]
The nice thing is, the arcade stick itself can be used wired or wireless on other devices, which means it will become my go to fighter/arcade stick on my MiSTer...
